src/assets/svg/spinner.svg

```diff
@@ -1,17 +1,7 @@
-    <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <mask id="mask">
-        <circle cx="12" cy="12" r="10" stroke="white" stroke-width="2" />
-        <rect width="12" height="12" fill="black" stroke-width="0" />
-        <circle cx="2" cy="12" r="1" fill="white" stroke-width="0" />
-        <circle cx="12" cy="2" r="1" fill="white" stroke-width="0" />
-      </mask>
-      <circle
-        cx="12"
-        cy="12"
-        r="10"
-        stroke-width="2"
-        stroke-linecap="round"
-        stroke-linejoin="round"
-        mask="url(#mask)"
-      />
-    </svg>
\ No newline at end of file
+<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why this change? The spinner won't render on GitHub, but the current state is explicitly a circle, while the new state is a path, so it may render worse if magnified.

Reply:
it is a new image, so it's worth the change. it _could_ be implemented as a more human-readable svg, but is not worth the effort in this case.